shard-operator-parent/shard-operator-v1/src/main/java/com/redhat/service/smartevents/shard/operator/v1/resources/istio/requestauthentication/RequestAuthentication.java

```diff
@@ -1,4 +1,4 @@
-package com.redhat.service.smartevents.shard.operator.resources.istio.requestauthentication;
+package com.redhat.service.smartevents.shard.operator.v1.resources.istio.requestauthentication;
```

Comment:
`requestauthentication` could be moved to the `core`

Reply:
Done. I've moved `knative` and `istio` to `-core`.